CONTRIBUTING.md

```diff
@@ -37,9 +37,17 @@ Before submitting a bug report, please follow the checklist to help us address t
     - Attach your input and output, if feasible.
     - Attach the stack trace.
     - Make note of your OS, Platform and Version.
-    - Make not of your environment configs (compiler, package manager, conda env dependencies)
+    - Make note of your environment configs (compiler, package manager, conda env dependencies)
     - Verify that the bug is reproducible.
 
+## Requesting Documentation
+
+When requesting new documentation, the author of the issue should specify both the topic and category of the desired documentation.  The documentation category must be "Getting Started," "How-To," "Concept," or "Manual," and the specific requirements for posting the issue will be determined by the documentation category.  Each of the documentation categories has an associated issue template that will guide the requester through the following category-specific requirements:
+    - Getting Started: a lesson or list of lessons to be learned (e.g. how does map projection with CSM work in ISIS) and an associated task by which those lessons will be conveyed (e.g. create a map-projected image using CSM).
+    - How-To: a definition of the basic problem that needs to be solved (e.g. how to append to the CSM search directories)
+    - Concept: a pending title and the scope of the proposed documentation
```

Comment:
We should also insist in adding any source material. Basically what their sources will be. Moved from X, a summary of paper or abstract Y, etc.